hs-abci-examples/nameservice/src/Nameservice/Modules/Token.hs

```diff
@@ -62,6 +70,16 @@ instance HasCodec Amount where
     encode (Amount b) = cs $ Binary.encode b
     decode = Right . Amount . Binary.decode . cs
 
+-- Address orphans
```

Comment:
if you add orphan instances you should use `{-# OPTIONS_GHC -fno-warn-orphans #-} pragam at the top